src/Layout/redist/targets/OverlaySdkOnLKG.targets

```diff
@@ -25,10 +25,13 @@
       </TemplatesFolderPath>
     </ItemGroup>
     <Error Text="SDK Stage 0 has more than one folder with templates: @(TemplatesFolderPath->'%(Identity)'). Please delete all but one and rebuild" Condition="@(TemplatesFolderPath->Count()) > 1"></Error>
+    <PropertyGroup>
+      <TemplateDestinationPath>$(RedistLayoutPath)templates\$([System.IO.Path]::GetFileName(@(TemplatesFolderPath)))</TemplateDestinationPath>
+    </PropertyGroup>
 
     <!--Prepare Microsoft.DotNet.Common.*.nupkg and pack them directly to target <redist root>\templates\<runtime version> folder. -->
-    <Exec Command="$(DotnetTool) pack $(RepoRoot)template_feed\Microsoft.DotNet.Common.ProjectTemplates.8.0 --configuration $(Configuration) --output $(RedistLayoutPath)\templates\@(TemplatesFolderPath->'%(FolderName)')\" />
-    <Exec Command="$(DotnetTool) pack $(RepoRoot)template_feed\Microsoft.DotNet.Common.ItemTemplates --configuration $(Configuration) --output $(RedistLayoutPath)\templates\@(TemplatesFolderPath->'%(FolderName)')\"  />
+    <MSBuild Projects="$(RepoRoot)template_feed\Microsoft.DotNet.Common.ProjectTemplates.8.0\Microsoft.DotNet.Common.ProjectTemplates.8.0.csproj" Targets="Pack" Properties="Configuration=$(Configuration);OutputPath=$(TemplateDestinationPath)" />
+    <MSBuild Projects="$(RepoRoot)template_feed\Microsoft.DotNet.Common.ItemTemplates\Microsoft.DotNet.Common.ItemTemplates.csproj" Targets="Pack" Properties="Configuration=$(Configuration);OutputPath=$(TemplateDestinationPath)" />
```

Comment:
Calling MSBuild targets here means we don't invoke dotnet CLI overhead twice unnecessarily and that we can take advantage of any build caching from the projects already